packages/nodes-base/nodes/Airtop/actions/extraction/query.operation.ts

```diff
@@ -17,7 +17,7 @@ export const description: INodeProperties[] = [
 		},
 		required: true,
 		default: '',
-		placeholder: 'Is there a login modal on this page?',
+		placeholder: 'e.g. Is there a login form in this page?',
```

Comment:
Let's add this same placeholder to `packages/nodes-base/nodes/Airtop/actions/extraction/getPaginated.operation.ts` since it currently doesn't have a placeholder.
<img width="385" alt="image" src="https://github.com/user-attachments/assets/3542c760-de5a-4e7c-ac2d-0a4471b085c1" />